logger/glog/glog_test.go

```diff
@@ -735,6 +737,86 @@ func TestParseInterval(t *testing.T) {
 	}
 }
 
+const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+const (
+	letterIdxBits = 6                    // 6 bits to represent a letter index
+	letterIdxMask = 1<<letterIdxBits - 1 // All 1-bits, as many as letterIdxBits
+	letterIdxMax  = 63 / letterIdxBits   // # of letter indices fitting in 63 bits
+)
+
+/*
+// Following code is commented-out because the test depends very strongly on
+// time and performance. On my (tzdybal) machine currently hardcoded parameters
```

Comment:
Think it would be worth using a `time.Sleep` stub here? 

I generally don't love to have commented code/tests rolling in... but also appreciate that the last thing we need is another running test that actually waits for a timeout.

Reply:
We need quite complicated time relations in here. The desired timing is following:
1. First invocation of `rotateOld` is scheduled and acquires list of log files.
2. Second invocation of `rotateOld` also starts execution and acquires list of log files.
3. One of the `rotateOld` executions finishes compression and the file is renamed.
4. The other execution tries to access the file that is already processed, causing Fatal.

Sleep is only used in test, and was adjusted empirically. We can try replace sleep with monitoring of changes in log directory, and also make files smaller to speedup the test.